package/cpp/api/JsiSkImage.h

```diff
@@ -125,6 +125,34 @@ class JsiSkImage : public JsiSkWrappingSkPtrHostObject<SkImage> {
     return jsi::String::createFromAscii(runtime, buffer);
   }
 
+  JSI_HOST_FUNCTION(readPixels) {
+    auto image = getObject();
+    if (image->isTextureBacked()) {
+      image = image->makeNonTextureImage();
```

Comment:
This is ok for now, I will investigate a bit further what are the different workflows there.
This is something we meant to do for a while.
Currently we let Skia handle uploading/removing images to the GPU but we may want to offer something more explicit there.

Reply:
I use copilot, it generate these code.
Seems because the `encodeToBase64` has this workflow.
I'm not sure is this required.